### PR TITLE
Expectation has procedure usage

### PR DESCRIPTION
**Pull request by faloi**
Refs uqbar-project/mumuki-gobstones-server#19

### CODE REVIEW

File: lib/gobstones/expectations/has_procedure_usage.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require_relative '../with_tempfile'
+
+module Gobstones::Expectations
+  class HasProcedureUsage
+    include Gobstones::WithTempfile
```
Comment (flbulgarelli) [inline lib/gobstones/expectations/has_procedure_usage.rb]: where do you use this included code?

Comment (faloi) [inline lib/gobstones/expectations/has_procedure_usage.rb]: I don't... in the intermediate version the tempfile was created here, and I forgot to delete it.

File: spec/gobstones/expectations/has_procedure_usage_spec.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+require_relative '../../../lib/gobstones'
+
+include Gobstones::Expectations
+
+describe HasProcedureUsage do
+  context '#value' do
+    let(:runner) { ExpectationsRunner.new('python .heroku/vendor/pygobstones/language/vgbs/gbs.py', 'program { IrAlBorde(Oeste) }') }
+    let(:results) { runner.run! [HasProcedureUsage.new('IrAlBorde'), HasProcedureUsage.new('Mover')] }
+    it { expect(results.first[1]).to be_true }
```
Comment (flbulgarelli) [inline spec/gobstones/expectations/has_procedure_usage_spec.rb:11]: Ok, I think it is for now. But just I would like to notice that in te future, to be fully compatible with mumuki expectations, all expectations should have a binding they refer to.

That means, you are not going to check if a procedure is used, but if a procedure is used within anothe

File: lib/gobstones/expectations/expectations_runner.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+require_relative '../with_tempfile'
+
+module Gobstones::Expectations
+  class ExpectationsRunner
+    include Gobstones::WithTempfile
+
+    def initialize(gobstones_command, source_code)
+      @gobstones_command = gobstones_command
+      @source_code = source_code
```
Comment (flbulgarelli) [inline lib/gobstones/expectations/expectations_runner.rb:9]: Perhaps you could use here Gobstones::Language::Gobstones.gobstones_command

File: spec/gobstones/expectations/has_procedure_usage_spec.rb (hunk 2 above)
Comment (faloi) [inline spec/gobstones/expectations/has_procedure_usage_spec.rb:11]: Agree, could you submit an issue?

File: lib/gobstones/expectations/expectations_runner.rb (hunk 3 above)
Comment (faloi) [inline lib/gobstones/expectations/expectations_runner.rb:9]: Could be, but that conflicts with
https://github.com/uqbar-project/gobstones-spec/issues/2

Comment (flbulgarelli) [inline lib/gobstones/expectations/expectations_runner.rb:9]: OK, good point

**Pull request merged**